### PR TITLE
tests: remove some trailing ws

### PR DESCRIPTION
**Pull request by klensy**
Cleans one more case of trailing whitespace in tests.

### CODE REVIEW

**Comment by klensy** [comment]
@rustbot author

Blessing clippy & rustfmt

File: compiler/rustc_errors/src/emitter.rs [hunk 1]
```diff
@@ -2260,13 +2260,18 @@ impl HumanEmitter {
...
                 [] => {
-                    draw_col_separator(buffer, *row_num, max_line_num_len + 1);
+                    draw_col_separator_no_space(buffer, *row_num, max_line_num_len + 1);
                 }
                 _ => {
                     buffer.puts(*row_num, max_line_num_len + 1, "~ ", Style::Addition);
                 }
             }
-            buffer.append(*row_num, &normalize_whitespace(line_to_add), Style::NoStyle);
+            buffer.puts(
```
Comment (compiler-errors) [inline compiler/rustc_errors/src/emitter.rs:2269]: What's the difference here (specifically, why do we need `+ 3`)?  Would be worth adding comments explaining a lot of the logic for the `+ N`s in this function.

Comment (klensy) [reply compiler/rustc_errors/src/emitter.rs:2274]: Added explanation for that changed part.

**Review by compiler-errors — APPROVED** [review]

**Pull request merged**